src/components/SetupApplication/SetupApplicationForm.tsx

```diff
@@ -36,13 +36,27 @@ export const SetupApplicationForm: FC<Props> = ({
             openMonth: openDate.getMonth() + 1 || "",
             openDay: openDate.getDate() || "",
             openHour: openDate.getHours() || "",
-            openMinute: openDate.getMinutes() || "",
+            openMinute:
```

Comment:
Can you refactor this. These sort of nested ternaries are really hard to read, and it seems a very convulated way to just zero-pad a number.

Reply:
I've moved it into a function